compiler/crates/relay-compiler/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use env_logger::Env;
+use log::{error, info};
+use relay_compiler::{compiler::Compiler, config::Config};
+use std::{path::PathBuf, sync::Arc};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(
+    name = "Relay Compiler",
+    about = "Compiler to produce Relay generated files."
+)]
+struct Opt {
+    /// Compile and watch for changes
+    #[structopt(long, short)]
+    watch: bool,
+
+    /// Path to the compiler config file
+    config: PathBuf,
+
+    /// Root directory of the project
+    root: PathBuf,
+}
+
+#[tokio::main]
+async fn main() {
+    env_logger::from_env(Env::default().default_filter_or("info, error, warn")).init();
```

Comment:
I feel like we should pull in a different printer so its _pretty_? And potentially scope the errors also?

Reply:
Yeah, this is still a pretty rough edge.

I see 2 stages:
- Clean this up for end users to be more to the essentials, probably update some messages and the formatting to remove timestamps.
- I dream of something like the Jest test runner / yarn install to show progress (probably more important internally if it takes a couple seconds even with Rust) and just generally nicer output.

For now this is basically debug output level for development.